packages/service-library/src/servicelib/logging_utils.py

```diff
@@ -79,20 +86,18 @@ def config_all_loggers():
     loggers = [logging.getLogger()] + [
         logging.getLogger(name) for name in the_manager.loggerDict
     ]
+
+    FINAL_FORMATTING = DEFAULT_FORMATTING + f" | log_service={service_name}"
```

Comment:
still wondering if this is really necessary.
use-case 1: I do docker log/portainer: I know which service I'm in
use-case 2: Graylog: it knows the container name / id already. does this really brings any thing more?

Reply:
Ok, I do not need to complicate our Python code with this so I will remove it. But I will introduce an extractor that will parse the container name -> to service name in Graylog. I see it as really handy when you want to see logs on the level of service and not a container and it will save you always some time so you do not need to regex. Additional benefit might be that when you want to construct for example heatmap of errors and services, ones again it will be handy that you have these two fields already predefined.